tests/cli/multiple-files.test

```diff
@@ -58,3 +58,11 @@ hledger print -f personal.journal -f ../journal/a.timeclock -f ../journal/b.time
 
 >>>2
 >>>=0
+
+# 4. same-date transactions in different files are shown in the right order
+hledger -f a.j -f b.j reg
```

Comment:
I have tried to avoid creating the two extra files `a.j` and `b.j` by doing this:
```
echo -e '2018/1/1 a1\n (a)  1' >> a.j; echo -e '2018/1/1 b1\n (b)  1' >> b.j; hledger -f a.j -f b.j reg; rm a.j b.j
```
But I got an error because it was running a wrong command:
```
Command:
/full_path/hledger  -e '2018/1/1 a1\n (a)  1' >> a.j; echo -e '2018/1/1 b1\n (b)  1' >> b.j; hledger -f a.j -f b.j reg; rm a.j b.j
```

Reply:
shelltest's -w the first word of the command; you can prefix it with a space to prevent that. Also it's good to create such temp files with $$ in the name for concurrency safety, you'll see some other examples. But I think the way you did it is better, simple and more portable.